e3nn/rs.py

```diff
@@ -375,7 +384,11 @@ def tensor_product(input1, input2, output, normalization='component', sorted=Fal
 
     if isinstance(input2, list) and isinstance(output, list):
         Rs_in1, Q = _tensor_product_in_out(input2, input1, output, normalization, sorted)
-        Q = torch.einsum('kij->kji', Q)
+        # Q = torch.einsum('kij->kji', Q)
```

Comment:
Thanks, it helped me to spot a mistake.
Always be careful to specify `sparse_sizes` when creating a `SparseTensor`. Because it will infer it but you could have an extra column or row of zeros that it cannot infer.